Extensions/PrimitiveDrawing/JsExtension.cpp

```diff
@@ -42,6 +42,63 @@ class PrimitiveDrawingJsExtension : public gd::PlatformExtension {
     GetAllActionsForObject(
         "PrimitiveDrawing::Drawer")["PrimitiveDrawing::Line"]
         .SetFunctionName("drawLine");
+
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::LineV2"]
+        .SetFunctionName("drawLineV2");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::Ellipse"]
+        .SetFunctionName("drawEllipse");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::RoundedRectangle"]
+        .SetFunctionName("drawRoundedRectangle");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::Star"]
+        .SetFunctionName("drawStar");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::Arc"]
+        .SetFunctionName("drawArc");
+/*
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::ArcTo"]
+        .SetFunctionName("drawArcTo");
+*/
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::BezierCurve"]
+        .SetFunctionName("drawBezierCurve");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::QuadraticCurve"]
+        .SetFunctionName("drawQuadraticCurve");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::BeginFillPath"]
+        .SetFunctionName("beginFillPath");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::EndFillPath"]
+        .SetFunctionName("endFillPath");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::PathMoveTo"]
+        .SetFunctionName("drawPathMoveTo");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::PathLineTo"]
+        .SetFunctionName("drawPathLineTo");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::PathBezierCurveTo"]
+        .SetFunctionName("drawPathBezierCurveTo");
+    GetAllActionsForObject(
+        "PrimitiveDrawing::Drawer")["PrimitiveDrawing::PathArc"]
```

Comment:
So this name "PathArc" seems inconsistent with the others ("PathLineTo"). But there is already the commented "PathArcTo". So I think we should remove "To" from others actions (PathLineTo => PathLine).

Reply:
Well actually this inconsistency is in Pixi.js too, so let's keep it.